Contracts/TokenAuth Api/request.json

```diff
@@ -0,0 +1,7 @@
+//GET /api/tokenauth
```

Comment:
is this call validates token or gets the token against auth key .?
what is the use case of this api